Packages/com.unity.inputsystem/Documentation~/Debugging.md

```diff
@@ -28,41 +23,45 @@ The debugger shows a tree breakdown of the state of the input system.
 |Layouts|A breakdown of all registered control and device layouts. This is essentially the database of supported hardware and the knowledge of how to represent a given piece of input hardware.|
 |Actions|Only visible in play mode and only if there are [actions](Actions.md) that are currently enabled.<br><br>Shows the list of all currently enabled actions and the controls they are bound to.<br><br>See [Debugging Actions](#debugging-actions).|
 |Users|Only visible when there is one or more `InputUser` instances. See [documentation](UserManagement.md).<br><br>Lists all currently active uers along with their active control schemes and devices, all their associated actions as well as the controls they are bound to.<br><br>Note that `PlayerInput` uses `InputUser` underneath. This means that when using `PlayerInput` components, each player will have an entry here.<br><br>See [Debugging Users/PlayerInput](#debugging-usersplayerinput).|
-|Settings||
-|Metrics||
+|Settings|Shows the currently active Input System [settings](Settings.md).|
+|Metrics|Shows some statics about Input System resource usage.|
 
-### Debugging Remotely
+### Debugging Devices
 
-You can connect input debugger to a player running on a device. This makes input activity from the player observable in the editor. The mechanism used by this is Unity's `PlayerConnection`, i.e. the same mechanism by which the Unity profiler can be connected to a player.
+You can double-click on any [input device](Devices.md) in the `Devices` list in the debugger window to open a window showing information about the device, including real-time state information for its controls.
```

Comment:
These days there's also the option to right-click a device. Gives some additional options.

Reply:
I actually disagree with this. While a good UI should implement all commonly expected gestures to interact with an item, I don't think a good documentation should always list  all possible paths to get there (at least not when the other paths don't add useful functionality unique to those paths). IMO that makes the text harder to read, and does not add much useful information.

Reply:
Sure, no problem with that :)